examples/todoapp/lib/base/app/config/app_constants.dart

```diff
@@ -11,4 +11,9 @@
 
 /// Vapid Key is required in order to for FCM to run on web properly
 /// https://github.com/FirebaseExtended/flutterfire/blob/4c9b5d28de9eeb5ce76c856fbd0c7b3ec8615e45/docs/messaging/usage.mdx#web-tokens
+
+import 'is_in_test_mode_io_not_available.dart' if (dart.library.io) 'is_in_test_mode_io_available.dart' if (dart.library.html) 'is_in_test_mode_io_not_available.dart';
+
 const String webVapidKey = '';
+
+bool isInTestMode = isInTestModePlatformSpecific;
```

Comment:
Do we really need to have two global variables for the same thing ?

Reply:
Isn't it pointing to the same refference? It's to encourage the developers to use the one in the `app_constants`. Otherwise they will need to make the conditional import in every file they need this variable and the IDE will not suggest that, but will suggest to import the variable form one of the platform specific files.

<img width="841" alt="image" src="https://github.com/Prime-Holding/rx_bloc/assets/165766690/02981095-b89f-4626-a173-dc7ec038b3be">

Reply:
I believe you can do the following, and it will work just as before

1. Rename the isInTestModePlatformSpecific to isInTestMode
2. Get rid of the global variable in the app_constants.dart

Reply:
Done